src/app/globals.css

```diff
@@ -54,21 +54,30 @@ body {
   color: var(--text-color-primary);
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100%;
+  background: linear-gradient(
+    150deg,
+    rgba(201, 212, 242, 1) 3%,
+    rgba(228, 247, 255, 1) 40%
+  );
+  background-repeat: no-repeat;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
 
-#app {
-  height: 100vh;
-}
-
 h3 {
   font-family: 'Roboto', sans-serif;
   color: #f22;
   font-size: 1rem;
   font-weight: 300;
   text-align: center;
 }
+
+.article {
+  background-color: var(--card-bg);
+  border-radius: var(--border-radius);
+  box-shadow: 0 7px 12px -7px rgba(20, 83, 118, 0.39);
+}
```

Comment:
Move this to the `DoctorProfile.module.css`

Reply:
done. Thank you!